<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgQuickRoomAssignment</class>
 <widget class="QDialog" name="DlgQuickRoomAssignment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quick room assignment</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="EQLineEdit" name="letoRoom">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="lbto">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/images/remarks.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="EQLineEdit" name="letoGuest">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Entry</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="EDateEdit" name="deEntry">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="EQLineEdit" name="lefrRoom">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="EDateEdit" name="deDeparture">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>From room</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Departure</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>To room</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lbfr">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/images/remarks.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="EQLineEdit" name="lefrGuest">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Reservatin code</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leReserve">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>213</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>213</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="leInvoice">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lefrRoom</tabstop>
  <tabstop>letoRoom</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
